SWLOR.Game.Server/Service/CraftService/RecipeCategoryType.cs

```diff
@@ -54,7 +54,7 @@ public enum RecipeCategoryType
         Rifle = 23,
         [RecipeCategory("Twin Blade", true)]
         TwinBlade = 24,
-        [RecipeCategory("Bed", true)]
+        [RecipeCategory("Beds", true)]
```

Comment:
This is really minor but all of the recipe categories are singular for consistency sake. Can you swap this back to "Bed" and rename "Walls" to "Wall" below?

Reply:
note to self: done